`gui/mod-manager`: Add a window showing the active mods in a world (#1431)
DFHack/scripts: docs/gui/mod-manager.rst

===============

.. dfhack-tool::
    :summary: Manange your active mods.
    :tags: dfhack interface

In a loaded world, shows a list of active mods with the ability to copy to clipboard.

Usage
-----

::

    gui/mod-manager

Overlay
-------

> This tool also provides one overlay that is managed by the `overlay`

Author: ```suggestion
This tool also provides two overlays that are managed by the `overlay`
```
